src/bundle.rs

```diff
@@ -23,16 +23,16 @@ use sxd_xpath::{Context, Factory};
 
 use std::fs::File;
 use std::io::Read;
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
 pub enum BuildType {
     Unknown,
     Product,
     Bundle,
 }
 
-pub fn get_build_type(wxiobj: &Path) -> Result<BuildType> {
+pub fn get_build_type_for_one(wxiobj: &Path) -> Result<BuildType> {
```

Comment:
Could this method be refactored into a [`TryFrom`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) implementation on the `BuildType` enum and then used within the `get_build_type` function?

Roughly:

```rust
impl<'a> TryFrom<&'a Path> for BuildType {
    ...
    fn try_from(p: &'a Path) -> Result<Self> {
         ...
    }
}
```